gbpservice/nfp/configurator/agents/generic_config.py

```diff
@@ -268,8 +279,9 @@ def _process_event(self, ev):
             if (resource_data.get('periodicity') == gen_cfg_const.INITIAL and
                     result == common_const.SUCCESS):
                 notification_data = self._prepare_notification_data(ev, result)
-                self.sc.poll_event_done(ev)
+                # self.sc.poll_event_done(ev)
                 self.notify._notification(notification_data)
+                return STOP_POLLING
```

Comment:
Didn't get the difference between returning STOP_POLLING vs invoking poll_event_done. Is this functionality fix or perf?